### ISSUE TITLE
Create Account not working on mobile and tablet devices

### ISSUE THREAD

**Issue by monicaholloway** [tomatocart/TomatoCart-v1#254]
I am unable to create an account on any mobile or tablet devices. The screen freezes up and you can't enter any information on the page. Works fine on laptops.

**Comment by martin03** [comment]
Reported that the iPhone when creating a profile screen data entry freezes and does not allow recording of text.

**Comment by dart-com-ua** [comment]
yes, agree, 1.1.8.6.1 and account can't be created from mobile devices

**Comment by Sonyphile** [comment]
Ok I have a work around seeing as there is no official answer to this.

Open -  /templates/bootstrap/content/account/create.php

Goto line 172 and comment out the script as follows: -



In the admin panel click start->configuration->configuration->regulations and set privacy notice to false.

Account creation now possible on all mobile platforms.

Hope for more support soon from tomatocart or I may have to jump ship :(

**Comment by martin03** [comment]
How to comment on those lines?

**Comment by gregdawn** [comment]
Trying to follow the instructions to get it to work. Sonyphile, on your instructions to goto line 172 and comment out the script. I want to make sure that I am on the correct line and scripting it as needed. Do you happen to still have the original line and what you scripted it out to?

**Comment by martin03** [comment]
line 172 <script type="text/javascript">
    /*window.addEvent("domready",function() {
        var overlay = new Overlay(); 
        var box = new MultiBox('multibox', { 
            overlay: overlay
        });
    });*/
</script>
